### DISCUSSION TITLE
How is Whoogle a metasearch engine if the only search it uses is Google?

### DISCUSSION THREAD

**Discussion by kiasoc5** [benbusby/whoogle-search#1112] [Q&A]
SearxNG is a metasearch engine that uses several engines, but Whoogle afaik only uses one engine (Google).

I saw https://github.com/benbusby/whoogle-search/issues/123#issuecomment-691170674 but it's still unclear.

**Comment by WindyLibra** [comment]
Metasearch engine uses other search indexes, not engines in particular.
Just search engines like Google, Bing, Mojeek and partly Brave use their own search indexes.

So even DuckDuckGo and Ecosia are metasearch engines because of usage of other search engines, even if it's only one engine.
And knowing that, Whoogle is also a metasearch engine.

I hope I explained it well.
